### DISCUSSION TITLE
How to set a gauge  collecting data by second

### DISCUSSION THREAD

**Discussion by william-peng-cn** [apache/skywalking#10268] [Q&A] [answered]
It seems, for a gauge you can only see data by time granularity of minute. Then how to set a gauge  collecting data by second?
In the code of TomcatThreadExecutorInterceptor.java or Gauge.java，it also seems no field to configure.
Thanks!

![image](https://user-images.githubusercontent.com/48680780/211963872-bd6898e3-0332-4a43-9f0b-393bd2378dca.png)
![image](https://user-images.githubusercontent.com/48680780/211963442-44e7dd8e-53d7-4cbd-a1df-879d80ca1b19.png)

**Comment by wu-sheng** [comment, marked as answer]
No, there isn't. It costs too much(resource) in the product if you use the minute dimension.

**Reply by william-peng-cn** [reply]
thanks！
